arm/Microsoft.EventHub/namespaces/deploy.bicep

```diff
@@ -246,7 +225,6 @@ module eventHubNamespace_rbac '.bicep/nested_rbac.bicep' = [for (roleAssignment,
   name: '${uniqueString(deployment().name, location)}-EventHubNamespace-Rbac-${index}'
```

Comment:
I'd suggest to align the name to `name: '${deployment().name}-rbac-${index}'`

Reply:
Fixed